src/wasm/flb_wasm.c

```diff
@@ -71,7 +71,7 @@ static int flb_wasm_load_wasm_binary(const char *wasm_path, int8_t **out_buf, ui
 
 error:
 
-    return -1;
+    return FLB_FALSE;
```

Comment:
This return value will be checked by following if statement at `flb_wasm_instantiate`.
`-1` will be true in this case.
```c
    if(!flb_wasm_load_wasm_binary(wasm_path, &buffer, &buf_size)) {
        goto error;
    }
```